dev/tools/localization/gen_l10n.dart

```diff
@@ -673,7 +681,7 @@ class LocalizationsGenerator {
       }
 
       final Message message = Message(bundle, key);
-      if (pluralValueRE.hasMatch(message.value))
+      if (message.isPlural)
```

Comment:
I think we can remove line 357 `static RegExp _pluralRE...`, since it's not being used anywhere else

Reply:
Oops, thanks for catching that!